lab-said/README.md

```diff
@@ -0,0 +1,28 @@
+#### Feature Tasks  
```

Comment:
Need a completely original readme instead of a modified version of the lab readme.